### ISSUE TITLE
Batch norm and related Operator

### ISSUE THREAD

**Issue by JiayiFeng** [PaddlePaddle/Paddle#3658]

**Comment by Canpio** [comment]
1. Survey 8/25 Friday EOD
  - Tensorflow
  - Caffe2
